documentation/src/main/asciidoc/stories/assembly_cache_modes.adoc

```diff
@@ -8,6 +8,25 @@
 * Optimizing databases by decreasing the volume of write operations.
 * Providing resiliency and durability for consistent data across clusters.
 
+[IMPORTANT]
+====
+{brandname} places a restriction of less than or equal to 255 characters on a cache name or a cache template name.
+
+If you exceed this character limit, {brandname} throws an exception.
+
```

Comment:
Why did you add these empty lines inside the admonitions?

Reply:
From my experience on other projects, I do this to indicate paragraphs. Might I ask the impact of having blank lines? Do things break?